src/App.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list